src/homepage/_posts/2020-04-06-james-3.5.0.markdown

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title:  "Apache James Server 3.5.0"
+date:   2020-04-06  15:16:30 +0200
+categories: james update
+---
+
+The Apache James developers are pleased to announce James server 3.5.0 release.
+
+Here are some points we worked on:
+
+### Added
+- Distributed task management for Guice cassandra-rabbitmq product. This enables several James servers to share a consistent view
```

Comment:
could we stop talking about cassandra-rabbitmq product and start talking about "Distributed James" instead?